### DISCUSSION TITLE
Support for Coinpanda (tax service)

### DISCUSSION THREAD

**Discussion by itseivind** [pwillworth/dfkreport#29] [General]
Hi there!

Many of our users have requested us to be added to your website, so it would be great if that would be possible 🙏 I have sent dm on Discord with more details about our generic file format (username: eivind#6968)

Thanks in advance
/Eivind

**Comment by pwillworth** [comment]
Hi, sure I can add support for Coinpanda.  I do not see a message request in Discord from you.  Can you share the info here or a link to it maybe?

**Reply by itseivind** [reply]
The friend request says still pending on my side, but don't think I can share a direct link to it.
Can you try searching for eivind#6968 and see if you find me?

Btw I tried sending you a new message, but Discord says `Your message could not be delivered. This is usually because you don't share a server with the recipient or the recipient is only accepting direct messages from friends.`

**Reply by pwillworth** [reply]
weird, I don't see it.  I found you though and submitted a friend request to you.  Maybe requests don't show up if no servers shared?

Any reason why you can't share the info here?  Seems like most services publish the import format on their public site or user help docs.
